CHANGELOG.yaml

```diff
@@ -14,6 +14,14 @@ meta:
 
 changes:
 
+  - title: Prioritize `organ` and `organ_part` fields from Organoids over Specimens
+    issues:
+      - https://github.com/DataBiosphere/azul/issues/695
+      - https://github.com/humancellatlas/data-browser/issues/302
+    upgrade:
+      - reindex
+    notes: SpecimenVisitor now visits ancestors and descendants instead of just ancestors.
```

Comment:
This seems like an arbitrary implementation detail. The primary reason we maintain this change log is to document what other developers have to do to upgrade and to document externally visible changes. This details belongs in neither of those categories. Please remove.